api/src/controllers/settings.js

```diff
@@ -53,6 +53,8 @@ module.exports = {
       .then(() => {
         const replace = req.query && req.query.replace;
         const overwrite = req.query && req.query.overwrite;
+        // eslint-disable-next-line no-console
+        console.log(JSON.stringify(req.body, null, 2));
```

Comment:
Mistake?

Reply:
oh yes! I'll remove, thanks for noticing!